## [exact_001.gr](exact/exact_001.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
10 | 15 | 0.1 | named | https://github.com/freetdi/named-graphs | PetersenGraph.gr | 6 | 

**Description:** Petersen graph

![](vis/exact_001.png?raw=true)
## [exact_002.gr](exact/exact_002.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
16 | 58 | 0.3 | social | http://networkrepository.com/ | soc-tribes.gr | 11 | 

**Description:** Tribes is social network of tribes of the Gahuku–Gama alliance structure of the Eastern Central Highlands of New Guinea, from Kenneth Read (1954). The dataset contains a list of all of links, where a link represents signed friendships between tribes.

![](vis/exact_002.png?raw=true)
## [exact_003.gr](exact/exact_003.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
17 | 62 | 0.3 | biology | http://ginsim.org/models_repository | MAPK_red1_19062013.gr | 11 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains the most crucial components of the Mitogen-Activated Protein Kinase (MAPK) network in bladder cancer cells. Reference: Grieco, L., Calzone, L., Bernard-Pierrot, Isabelle, Radvanyi, Fran\c cois, Kahn-Perl\`es, Brigitte, & Thieffry, D. (2013). Integrative modelling of the influence of mapk network on cancer cell fate decision. PLOS Computational Biology, 9(10), 1003286. http://dx.doi.org/10.1371/journal.pcbi.1003286

![](vis/exact_003.png?raw=true)
## [exact_004.gr](exact/exact_004.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
18 | 18 | 0.1 | biology | http://ginsim.org/models_repository | VEGF_Pathway_12Jun2013_0.gr | 4 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the Drosophila VEGF signalling pathway. Reference: Mbodj, A., Junion, G., Brun, C., Furlong, E. E. M., & Thieffry, D. (2013). Logical modelling of drosophila signalling pathways. Molecular BioSystems, 9(9), 2248–2258. http://dx.doi.org/10.1039/C3MB70187E

![](vis/exact_004.png?raw=true)
## [exact_005.gr](exact/exact_005.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
20 | 21 | 0.1 | generated | Python networkx package | waxman_30.gr | 5 | 

**Description:** A Waxman graph (a random geometric graph) generated using waxman_graph function from Python’s networkx package

![](vis/exact_005.png?raw=true)
## [exact_006.gr](exact/exact_006.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
20 | 24 | 0.1 | computer science | https://github.com/freetdi/CFGs | contiki_mac_mac_call_sent_callback.gr | 5 | 

**Description:** A control-flow graph of a C function

![](vis/exact_006.png?raw=true)
## [exact_007.gr](exact/exact_007.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
20 | 30 | 0.2 | named | https://github.com/freetdi/named-graphs | FlowerSnark.gr | 9 | 

**Description:** A flower snark J_5, see https://en.wikipedia.org/wiki/Flower_snark

![](vis/exact_007.png?raw=true)
## [exact_008.gr](exact/exact_008.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
20 | 39 | 0.2 | planted | PACE 2020 | planted_rand_tree_20+20e.gr | 5 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_008.png?raw=true)
## [exact_009.gr](exact/exact_009.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
21 | 25 | 0.2 | infrastructure | Open street maps | Gortatowo_Poland.gr | 6 | 

**Description:** Gortatowo (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_009.png?raw=true)
## [exact_010.gr](exact/exact_010.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
21 | 72 | 0.6 | social | http://networkrepository.com/ | mammalia-vampire-bats-foodsharing.gr | 10 | 

**Description:** an animal social network

![](vis/exact_010.png?raw=true)
## [exact_011.gr](exact/exact_011.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
23 | 24 | 0.1 | biology | http://ginsim.org/models_repository | FGF_Pathway_12Jun2013.gr | 5 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains the nine key signalling pathways detected in Drosophila and recurrently used in metazoan development. Reference: Mbodj, A., Junion, G., Brun, C., Furlong, E. E. M., & Thieffry, D. (2013). Logical modelling of drosophila signalling pathways. Molecular BioSystems, 9(9), 2248–2258. http://dx.doi.org/10.1039/C3MB70187E

![](vis/exact_011.png?raw=true)
## [exact_012.gr](exact/exact_012.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
24 | 28 | 0.2 | infrastructure | Open street maps | Riga_Latvia.gr | 7 | 

**Description:** Riga (Latvia): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_012.png?raw=true)
## [exact_013.gr](exact/exact_013.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
25 | 46 | 0.4 | biology | http://networkrepository.com/ | ENZYMES_g172.gr | 7 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2006

![](vis/exact_013.png?raw=true)
## [exact_014.gr](exact/exact_014.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
25 | 52 | 0.3 | planted | PACE 2020 | planted_weighted_cycle_7_4+14v+14e.gr | 7 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_014.png?raw=true)
## [exact_015.gr](exact/exact_015.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
26 | 30 | 0.2 | biology | http://ginsim.org/models_repository | Wg_Pathway_11Jun2013.gr | 5 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the Drosophila Wg signalling pathway. Reference: Mbodj, A., Junion, G., Brun, C., Furlong, E. E. M., & Thieffry, D. (2013). Logical modelling of drosophila signalling pathways. Molecular BioSystems, 9(9), 2248–2258. http://dx.doi.org/10.1039/C3MB70187E

![](vis/exact_015.png?raw=true)
## [exact_016.gr](exact/exact_016.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
26 | 51 | 0.5 | biology | http://networkrepository.com/ | ENZYMES_g391.gr | 7 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2011

![](vis/exact_016.png?raw=true)
## [exact_017.gr](exact/exact_017.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
26 | 64 | 0.3 | biology | http://ginsim.org/models_repository | ModelT2DM_0.gr | 10 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This is a network of the signaling pathways that control S-phase entry and a specific type of senescence called geroconversion. Reference: Verlingue, L., Dugourd, Aur\'elien, Stoll, G., Barillot, E., Calzone, L., & Londo\~no-Vallejo, Arturo (2016). A comprehensive approach to the molecular determinants of lifespan using a boolean model of geroconversion. Aging Cell, 15(6), 1018–1026. http://dx.doi.org/10.1111/acel.12504

![](vis/exact_017.png?raw=true)
## [exact_018.gr](exact/exact_018.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
27 | 87 | 0.4 | biology | http://ginsim.org/models_repository | core_engine_budding_yeast_CC.gr | 11 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains the main documented interactions in budding yeast between core regulators of the cell cycle (Cdk/cyclins, APC, Cdk inhibitors). Reference: Faur\'e, Adrien, Naldi, Aur\'elien, Lopez, F., Chaouiya, C., Ciliberto, A., & Thieffry, D. (2009). Modular logical modelling of the budding yeast cell cycle. Molecular BioSystems, 5(12), 1787–1796. http://dx.doi.org/10.1039/B910101M

![](vis/exact_018.png?raw=true)
## [exact_019.gr](exact/exact_019.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
28 | 57 | 0.3 | biology | http://ginsim.org/models_repository | ap_boundary.gr | 7 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains the genes of the hedgehog signalling pathway of Drosophila melanogaster. Reference: Gonz\'alez, Aitor, Chaouiya, C., & Thieffry, D. (2008). Logical modelling of the role of the hh pathway in the patterning of the drosophila wing disc. Bioinformatics, 24(16), 234–240. http://dx.doi.org/10.1093/bioinformatics/btn266

![](vis/exact_019.png?raw=true)
## [exact_020.gr](exact/exact_020.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
28 | 168 | 0.9 | named | https://github.com/freetdi/named-graphs | JohnsonGraph_8_2.gr | 21 | 

**Description:** Johnson J(8,2) graph, see https://en.wikipedia.org/wiki/Johnson_graph

![](vis/exact_020.png?raw=true)
## [exact_021.gr](exact/exact_021.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
29 | 38 | 0.2 | computer science | https://github.com/freetdi/CFGs | fuzix_pathconf__pathconf.gr | 5 | 

**Description:** A control-flow graph of a C function

![](vis/exact_021.png?raw=true)
## [exact_022.gr](exact/exact_022.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
29 | 47 | 0.3 | biology | http://www.genome.jp/kegg/ | myb05169.kgml.gr-5.gr | 5 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "myb05160 Hepatitis C - Myotis brandtii (Brandt's bat)".

![](vis/exact_022.png?raw=true)
## [exact_023.gr](exact/exact_023.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
30 | 38 | 0.3 | infrastructure | Open street maps | Bierzyn_Poland.gr | 7 | 

**Description:** Bierzyn (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_023.png?raw=true)
## [exact_024.gr](exact/exact_024.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
30 | 45 | 0.3 | generated | Python networkx package | random-cubic_30.gr | 10 | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_024.png?raw=true)
## [exact_025.gr](exact/exact_025.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
30 | 51 | 0.5 | biology | http://networkrepository.com/ | ENZYMES_g378.gr | 8 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2010

![](vis/exact_025.png?raw=true)
## [exact_026.gr](exact/exact_026.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
30 | 59 | 0.3 | planted | PACE 2020 | planted_rand_tree_30+30e.gr | 5 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_026.png?raw=true)
## [exact_027.gr](exact/exact_027.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
30 | 70 | 0.4 | biology | http://ginsim.org/models_repository | Bladder_Model.gr | 11 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains human genes involved in tumorigenesis of bladder cells. Reference: Remy, E., Rebouissou, S., Chaouiya, C., Zinovyev, A., Radvanyi, Fran\c cois, & Calzone, L. (2015). A modeling approach to explain mutually exclusive and co-occurring genetic alterations in bladder tumorigenesis. Cancer Research, 75(19), 4042–4052. http://dx.doi.org/10.1158/0008-5472.CAN-15-0602

![](vis/exact_027.png?raw=true)
## [exact_028.gr](exact/exact_028.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
31 | 64 | 0.5 | biology | http://networkrepository.com/ | ENZYMES_g554.gr | 9 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2017

![](vis/exact_028.png?raw=true)
## [exact_029.gr](exact/exact_029.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
32 | 119 | 0.8 | social | http://networkrepository.com/ | aves-weaver-social-13.gr | 12 | 

**Description:** an animal social network. A network edge was drawn between individuals that used the same nest chambers either for roosting or nest-building at any given time within a series of observations at the same colony in the same year, either together in the nest chamber at the same time or at different times. Networks represent social data collected from 23 colonies of sociable weavers. From: van Dijk, Rene E., et al., "Cooperative investment in public goods is kin directed in communal nests of social birds." Ecology letters 17.9 (2014): 1141-1148.

![](vis/exact_029.png?raw=true)
## [exact_030.gr](exact/exact_030.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
32 | 137 | 0.8 | biology | http://ginsim.org/models_repository | SuppMat_Model_Master_Model.gr | 16 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes in the molecular pathways enabling tumour cell invasion and migration. Reference: Cohen, D. P. A., Martignetti, L., Robine, S., Barillot, E., Zinovyev, A., & Calzone, L. (2015). Mathematical modelling of molecular pathways enabling tumour cell invasion and migration. PLOS Computational Biology, 11(11), 1004571. http://dx.doi.org/10.1371/journal.pcbi.1004571

![](vis/exact_030.png?raw=true)
## [exact_031.gr](exact/exact_031.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
33 | 78 | 0.4 | biology | http://ginsim.org/models_repository | Collombet_model_Bcell_Macrophages_PNAS_170215.gr | 11 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in macrophage differentiation. Reference: Collombet, S., van Oevelen, C., Ortega, J. L. S., Abou-Jaoud\'e, Wassim, Stefano, B. D., Thomas-Chollier, Morgane, Graf, T., … (2017). Logical modeling of lymphoid and myeloid cell specification and transdifferentiation. Proceedings of the National Academy of Sciences, 114(23), 5792–5799. http://dx.doi.org/10.1073/pnas.1610622114

![](vis/exact_031.png?raw=true)
## [exact_032.gr](exact/exact_032.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
34 | 78 | 0.8 | social | http://www-personal.umich.edu/~mejn/netdata/ | karate_34.gr | 8 | 

**Description:** Zachary's karate club: social network of friendships between 34 members of a karate club at a US university in the 1970s. Please cite W. W. Zachary, An information flow model for conflict and fission in small groups, Journal of Anthropological Research 33, 452-473 (1977).

![](vis/exact_032.png?raw=true)
## [exact_033.gr](exact/exact_033.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
34 | 82 | 0.4 | biology | http://ginsim.org/models_repository | Th_differentiation_reduced_model.gr | 12 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in control of Th1/Th2/Th17/Treg cells differentiation. Reference: Naldi A, Carneiro J, Chaouiya C, Thieffry D (2010). Diversity and plasticity of th cell types predicted from regulatory network modelling. PLOS Computational Biology, 6(9), 1000912. http://dx.doi.org/10.1371/journal.pcbi.1000912

![](vis/exact_033.png?raw=true)
## [exact_034.gr](exact/exact_034.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
35 | 42 | 0.2 | computer science | https://github.com/freetdi/CFGs | contiki_shell-base64_base64_decode_char.gr | 7 | 

**Description:** A control-flow graph of a C function

![](vis/exact_034.png?raw=true)
## [exact_035.gr](exact/exact_035.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
35 | 70 | 0.6 | biology | http://networkrepository.com/ | ENZYMES_g109.gr | 9 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2004

![](vis/exact_035.png?raw=true)
## [exact_036.gr](exact/exact_036.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
37 | 68 | 0.6 | biology | http://networkrepository.com/ | ENZYMES_g427.gr | 6 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2014

![](vis/exact_036.png?raw=true)
## [exact_037.gr](exact/exact_037.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
39 | 48 | 0.3 | biology | http://www.genome.jp/kegg/ | sasa04530.kgml.gr-0.gr | 7 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "sasa04530 Tight junction - Salmo salar (Atlantic salmon)".

![](vis/exact_037.png?raw=true)
## [exact_038.gr](exact/exact_038.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
39 | 62 | 0.3 | biology | http://www.genome.jp/kegg/ | mun00071.kgml.gr-2.gr | 7 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "mun00071 Fatty acid degradation - Meriones unguiculatus (Mongolian gerbil)".

![](vis/exact_038.png?raw=true)
## [exact_039.gr](exact/exact_039.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
40 | 54 | 0.3 | biology | http://ginsim.org/models_repository | TCRsig40.gr | 7 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the TCR signalling pathway. Reference: Klamt, S., Saez-Rodriguez, Julio, Lindquist, J. A., Simeoni, L., & Gilles, E. D. (2006). A methodology for the structural and functional analysis of signaling and regulatory networks. BMC Bioinformatics, 7(1), 1–26. http://dx.doi.org/10.1186/1471-2105-7-56

![](vis/exact_039.png?raw=true)
## [exact_040.gr](exact/exact_040.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
40 | 73 | 0.6 | biology | http://networkrepository.com/ | ENZYMES_g274.gr | 9 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2008

![](vis/exact_040.png?raw=true)
## [exact_041.gr](exact/exact_041.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
40 | 79 | 0.4 | planted | PACE 2020 | planted_rand_tree_40+40e.gr | 5 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_041.png?raw=true)
## [exact_042.gr](exact/exact_042.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
40 | 112 | 0.6 | planted | PACE 2020 | planted_weighted_cycle_9_4+18v+18e.gr | 11 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_042.png?raw=true)
## [exact_043.gr](exact/exact_043.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
40 | 129 | 0.7 | biology | http://ginsim.org/models_repository | TCR-REDOX-METABOLISM_2019-07-26_reduced_0.gr | 14 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network captures the interplay between antigen recognition with reactive oxygen species and metabolic status in T cell responses. S\'anchez-Villanueva, Jos\'e Antonio, Rodr\'iguez-Jorge, Otoniel, Ram\'irez-Pliego, Oscar, Salgado, G. R., Abou-Jaoud\'e, Wassim,Hernandez, C\'eline, Naldi, Aur\'elien, … (2019). Contribution of ros and metabolic status to neonatal and adult cd8+ t cell activation. PLOS ONE, 14(12), 0226388. http://dx.doi.org/10.1371/journal.pone.0226388

![](vis/exact_043.png?raw=true)
## [exact_044.gr](exact/exact_044.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
41 | 52 | 0.3 | infrastructure | Open street maps | Ostrowite_Poland.gr | 7 | 

**Description:** Ostrowite (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_044.png?raw=true)
## [exact_045.gr](exact/exact_045.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
41 | 58 | 0.4 | generated | Python networkx package | expander-chordal-cycle_41.gr | 11 | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_045.png?raw=true)
## [exact_046.gr](exact/exact_046.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
41 | 76 | 0.6 | biology | http://networkrepository.com/ | ENZYMES_g418.gr | 7 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2013

![](vis/exact_046.png?raw=true)
## [exact_047.gr](exact/exact_047.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
41 | 256 | 1.6 | social | http://networkrepository.com/ | insecta-ant-trophallaxis-colony1.gr | 25 | 

**Description:** an animal social network. A trophallaxis event was recorded when ants engaged in mandible-to-mandible contact for greater than 1 s. Networks represent two C. pennsylvanicus colonies. Each colony was filmed for approximately 30 minutes for 8 consecutive nights. Citation: Quevillon, Lauren E., et al. "Social, spatial, and temporal organization in a complex insect society." Scientific reports 5 (2015).

![](vis/exact_047.png?raw=true)
## [exact_048.gr](exact/exact_048.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
42 | 67 | 0.4 | named | https://github.com/freetdi/named-graphs | WienerArayaGraph.gr | 13 | 

**Description:** Wiener-Araya graph, see https://en.wikipedia.org/wiki/Wiener%E2%80%93Araya_graph

![](vis/exact_048.png?raw=true)
## [exact_049.gr](exact/exact_049.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
42 | 73 | 0.4 | biology | http://ginsim.org/models_repository | RodriguezJorge_TLR5_Signalling_BooleanModel_17Jul2018.gr | 10 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the T cell receptor (TCR) signalling pathway and the Toll-like receptor (TLR5) signalling pathway. Reference: Rodr\'iguez-Jorge, Otoniel, Kempis-Calanis, Linda A., Abou-Jaoud\'e, Wassim, Guti\'errez-Reyna, Darely Y., Hernandez, C\'eline, Ramirez-Pliego, Oscar, Thomas-Chollier, Morgane, … (2019). Cooperation between t cell receptor and toll-like receptor 5 signaling for cd4+ t cell activation. Science Signaling, 12(577). http://dx.doi.org/10.1126/scisignal.aar3641

![](vis/exact_049.png?raw=true)
## [exact_050.gr](exact/exact_050.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
43 | 53 | 0.3 | computer science | https://github.com/freetdi/CFGs | fuzix_utent_getutid.gr | 8 | 

**Description:** A control-flow graph of a C function

![](vis/exact_050.png?raw=true)
## [exact_051.gr](exact/exact_051.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
45 | 91 | 0.7 | biology | http://networkrepository.com/ | ENZYMES_g596.gr | 11 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2018

![](vis/exact_051.png?raw=true)
## [exact_052.gr](exact/exact_052.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
45 | 630 | 3.5 | named | https://github.com/freetdi/named-graphs | KneserGraph_10_2.gr | 37 | 

**Description:** Kneser graph KG(10,2), see https://en.wikipedia.org/wiki/Kneser_graph

![](vis/exact_052.png?raw=true)
## [exact_053.gr](exact/exact_053.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
46 | 63 | 0.3 | biology | http://www.genome.jp/kegg/ | mun04621.kgml.gr-0.gr | 7 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "mun04621 NOD-like receptor signaling pathway - Meriones unguiculatus (Mongolian gerbil)".

![](vis/exact_053.png?raw=true)
## [exact_054.gr](exact/exact_054.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
47 | 67 | 0.4 | biology | http://ginsim.org/models_repository | Mast_cell_activation_Annotated_19oct2014.gr | 8 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains a molecular map for the FceRI and FcgRIIb signalling pathways, which play a key role in mast cell activation in mammals. Reference: Niarakis, A., Bounab, Y., Grieco, L., Roncagalli, R., Hesse, A., Garin, J\'er\^ome, Malissen, B., … (2014). Computational modeling of the main signaling pathways involved in mast cell activation. In M. Daeron, & F. Nimmerjahn (Eds.), Fc Receptors (pp. 69–93). Cham: Springer International Publishing.

![](vis/exact_054.png?raw=true)
## [exact_055.gr](exact/exact_055.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
49 | 85 | 0.5 | social | https://sites.google.com/site/ucinetsoftware/datasets | COCAINE_JUANES.gr | 9 | 

**Description:** A covert network. Operation JUANES (N=51). In 2009, the police investigation detected a group involved in the smuggling of cocaine from Mexico to be distributed in Madrid (Spain). In this case, the group operated in close cooperation with another organization that was laundering the illegal income from drug distribution from this and other groups. The cocaine traffickers earned an estimated EUR 60 million.

![](vis/exact_055.png?raw=true)
## [exact_056.gr](exact/exact_056.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
50 | 59 | 0.3 | biology | http://www.genome.jp/kegg/ | fpg00140.kgml.gr-4.gr | 7 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "fpg00140 Steroid hormone biosynthesis - Falco peregrinus (peregrine falcon)".

![](vis/exact_056.png?raw=true)
## [exact_057.gr](exact/exact_057.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
50 | 75 | 0.5 | generated | Python networkx package | random-cubic_50.gr | 13 | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_057.png?raw=true)
## [exact_058.gr](exact/exact_058.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
50 | 98 | 0.7 | biology | http://networkrepository.com/ | ENZYMES_g395.gr | 10 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2012

![](vis/exact_058.png?raw=true)
## [exact_059.gr](exact/exact_059.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
50 | 525 | 2.9 | named | https://github.com/freetdi/named-graphs | SwitchedSquaredSkewHadamardMatrixGraph_2.gr | 42 | 

**Description:** The result of SwitchedSquaredSkewHadamardMatrixGraph(5) in sage.

![](vis/exact_059.png?raw=true)
## [exact_060.gr](exact/exact_060.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
51 | 151 | 0.8 | planted | PACE 2020 | planted_weighted_cycle_11_4+22v+22e.gr | 12 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_060.png?raw=true)
## [exact_061.gr](exact/exact_061.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
53 | 100 | 0.6 | biology | http://ginsim.org/models_repository | MAPK_large_19june2013.gr | 13 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains the most crucial components of the Mitogen-Activated Protein Kinase (MAPK) network in bladder cancer cells. Reference: Grieco, L., Calzone, L., Bernard-Pierrot, Isabelle, Radvanyi, Fran\c cois, Kahn-Perl\`es, Brigitte, & Thieffry, D. (2013). Integrative modelling of the influence of mapk network on cancer cell fate decision. PLOS Computational Biology, 9(10), 1003286. http://dx.doi.org/10.1371/journal.pcbi.1003286

![](vis/exact_061.png?raw=true)
## [exact_062.gr](exact/exact_062.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
54 | 66 | 0.4 | infrastructure | Open street maps | Lubiewice_Poland.gr | 9 | 

**Description:** Lubiewice (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_062.png?raw=true)
## [exact_063.gr](exact/exact_063.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
55 | 100 | 0.8 | biology | http://networkrepository.com/ | ENZYMES_g598.gr | 12 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2019

![](vis/exact_063.png?raw=true)
## [exact_064.gr](exact/exact_064.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
57 | 79 | 0.6 | biology | http://networkrepository.com/ | ENZYMES_g541.gr | 8 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2016

![](vis/exact_064.png?raw=true)
## [exact_065.gr](exact/exact_065.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
57 | 153 | 0.9 | planted | PACE 2020 | planted_weighted_cycle_13_4+26v+26e.gr | 11 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_065.png?raw=true)
## [exact_066.gr](exact/exact_066.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
59 | 154 | 0.9 | biology | http://www.genome.jp/kegg/ | myi00240.kgml.gr-0.gr | 11 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "myi00240 Pyrimidine metabolism - Mizuhopecten yessoensis (Yesso scallop)".

![](vis/exact_066.png?raw=true)
## [exact_067.gr](exact/exact_067.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
60 | 72 | 0.5 | infrastructure | Open street maps | Chomiąża_Poland.gr | 9 | 

**Description:** Chomiąża (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_067.png?raw=true)
## [exact_068.gr](exact/exact_068.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
60 | 90 | 0.5 | generated | Python networkx package | random-cubic_60.gr | 17 | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_068.png?raw=true)
## [exact_069.gr](exact/exact_069.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
60 | 119 | 0.7 | planted | PACE 2020 | planted_rand_tree_60+60e.gr | 6 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_069.png?raw=true)
## [exact_070.gr](exact/exact_070.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
60 | 176 | 1.0 | biology | http://ginsim.org/models_repository | T_LGL.gr | 16 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in survival signaling in large granular lymphocyte leukemia. Reference: Zhang, R., Shah, M. V., Yang, J., Nyland, S. B., Liu, X., Yun, J. K., Albert, R\'eka, … (2008). Network model of survival signaling in large granular lymphocyte leukemia. Proceedings of the National Academy of Sciences, 105(42), 16308–16313.

![](vis/exact_070.png?raw=true)
## [exact_071.gr](exact/exact_071.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
61 | 151 | 0.8 | biology | http://www.genome.jp/kegg/ | ray00061.kgml.gr-0.gr | 15 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "ray00051 Fructose and mannose metabolism - Rousettus aegyptiacus (Egyptian rousette)".

![](vis/exact_071.png?raw=true)
## [exact_072.gr](exact/exact_072.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
62 | 108 | 0.8 | biology | http://networkrepository.com/ | ENZYMES_g199.gr | 10 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2007

![](vis/exact_072.png?raw=true)
## [exact_073.gr](exact/exact_073.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
62 | 159 | 1.7 | social | http://www-personal.umich.edu/~mejn/netdata/ | dolphins_62.gr | 16 | 

**Description:** Dolphin social network: an undirected social network of frequent associations between 62 dolphins in a community living off Doubtful Sound, New Zealand. Please cite D. Lusseau, K. Schneider, O. J. Boisseau, P. Haase, E. Slooten, and S. M. Dawson, Behavioral Ecology and Sociobiology 54, 396-405 (2003). Thanks to David Lusseau for permission to post these data on this web site.

![](vis/exact_073.png?raw=true)
## [exact_074.gr](exact/exact_074.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
63 | 1008 | 5.6 | named | https://github.com/freetdi/named-graphs | NonisotropicUnitaryPolarGraph_3_3.gr | 56 | 

**Description:** Graph NU(m=3,q=3): The graph on nonisotropic, with respect to a nondegenerate Hermitean form, points of the (m−1)-dimensional projective space over F_q, with points adjacent whenever they lie on a tangent (to the set of isotropic points) line.

![](vis/exact_074.png?raw=true)
## [exact_075.gr](exact/exact_075.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
64 | 168 | 1.0 | named | https://github.com/freetdi/named-graphs | HanoiTowerGraph_4_3.gr | 21 | 

**Description:** Hanoi Tower Graph for 3 disks and 4 towers

![](vis/exact_075.png?raw=true)
## [exact_076.gr](exact/exact_076.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
65 | 128 | 0.7 | biology | http://ginsim.org/models_repository | Th_differentiation_full_annotated_model.gr | 15 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in control of Th1/Th2/Th17/Treg cells differentiation. Reference: Naldi A, Carneiro J, Chaouiya C, Thieffry D (2010). Diversity and plasticity of th cell types predicted from regulatory network modelling. PLOS Computational Biology, 6(9), 1000912. http://dx.doi.org/10.1371/journal.pcbi.1000912

![](vis/exact_076.png?raw=true)
## [exact_077.gr](exact/exact_077.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
66 | 120 | 0.9 | biology | http://networkrepository.com/ | ENZYMES_g504.gr | 12 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2015

![](vis/exact_077.png?raw=true)
## [exact_078.gr](exact/exact_078.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
67 | 152 | 0.9 | planted | PACE 2020 | planted_weighted_cycle_17_4+34v+34e.gr | 10 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_078.png?raw=true)
## [exact_079.gr](exact/exact_079.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
68 | 83 | 0.5 | infrastructure | Open street maps | Różanna_Poland.gr | 9 | 

**Description:** Różanna (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_079.png?raw=true)
## [exact_080.gr](exact/exact_080.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
68 | 116 | 0.7 | social | https://sites.google.com/site/ucinetsoftware/datasets | ITALIAN_GANGS.gr | 8 | 

**Description:** A covert network: Italian gangs

![](vis/exact_080.png?raw=true)
## [exact_081.gr](exact/exact_081.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
70 | 79 | 0.4 | biology | http://www.genome.jp/kegg/ | psq01230.kgml.gr-0.gr | 7 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "psq01230 Biosynthesis of amino acids - Punctularia strigosozonata".

![](vis/exact_081.png?raw=true)
## [exact_082.gr](exact/exact_082.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
70 | 139 | 0.8 | planted | PACE 2020 | planted_rand_tree_70+70e.gr | 6 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_082.png?raw=true)
## [exact_083.gr](exact/exact_083.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
70 | 274 | 1.5 | social | http://konect.cc/ | highschool.gr | 22 | 

**Description:** This directed network contains friendships between boys in a small highschool in Illinois. Each boy was asked once in the fall of 1957 and the spring of 1958. This dataset aggregates the results from both dates. A node represents a boy and an edge between two boys shows that the left boy chose the right boy as a friend. The edge weights show how often that happened. As a boy could choose the same boy twice edge values from 1 to 2 are allowed.

![](vis/exact_083.png?raw=true)
## [exact_084.gr](exact/exact_084.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
71 | 103 | 0.6 | generated | Python networkx package | expander-chordal-cycle_71.gr | 16 | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_084.png?raw=true)
## [exact_085.gr](exact/exact_085.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
72 | 158 | 0.9 | biology | http://ginsim.org/models_repository | SP_6cells.gr | 13 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes related to the segment polarity genes engrailed and wingless in fly embryos. S\'anchez, Lucas, Chaouiya, C., & Thieffry, D. (2002). Segmenting the fly embryo: logical analysis of the role of the segment polarity cross-regulatory module. International Journal of Developmental Biology, 52(8), 1059–1075. http://dx.doi.org/10.1387/ijdb.072439ls

![](vis/exact_085.png?raw=true)
## [exact_086.gr](exact/exact_086.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
75 | 118 | 0.6 | biology | https://thebiogrid.org/ | Toxoplasma_gondii-Sibley2017.gr | 5 | 

**Description:** This is a gene-interaction network of Toxoplasma gondii, the most prevalent parasite in developed countries.

![](vis/exact_086.png?raw=true)
## [exact_087.gr](exact/exact_087.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
75 | 304 | 1.8 | social | Stanford Graph Base | huck.gr | 13 | 

**Description:** Huck graph from  Stanford Graph Base

![](vis/exact_087.png?raw=true)
## [exact_088.gr](exact/exact_088.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
77 | 146 | 0.8 | biology | http://ginsim.org/models_repository | Flobak_FullModel_S2_Dataset.gr | 16 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network accounts for cell fate decision network in the AGS gastric cancer cell line. Reference: Flobak, \AAsmund, Baudot, Ana\"is, Remy, E., Thommesen, L., Thieffry, D., Kuiper, M., & L\aegreid, Astrid (2015). Discovery of drug synergies in gastric cancer cells predicted by logical modeling. PLOS Computational Biology, 11(8), 1004426. http://dx.doi.org/10.1371/journal.pcbi.1004426

![](vis/exact_088.png?raw=true)
## [exact_089.gr](exact/exact_089.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
77 | 254 | 2.0 | social | http://www-personal.umich.edu/~mejn/netdata/ | lesmis_77.gr | 15 | 

**Description:** Les Miserables: coappearance network of characters in the novel Les Miserables. Please cite D. E. Knuth, The Stanford GraphBase: A Platform for Combinatorial Computing, Addison-Wesley, Reading, MA (1993).

![](vis/exact_089.png?raw=true)
## [exact_090.gr](exact/exact_090.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
77 | 616 | 3.5 | named | https://github.com/freetdi/named-graphs | M22Graph.gr | 55 | 

**Description:** M22 graph, see https://en.wikipedia.org/wiki/M22_graph

![](vis/exact_090.png?raw=true)
## [exact_091.gr](exact/exact_091.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
80 | 98 | 0.6 | infrastructure | Open street maps | Sienno_Poland.gr | 9 | 

**Description:** Sienno (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_091.png?raw=true)
## [exact_092.gr](exact/exact_092.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
80 | 120 | 0.7 | generated | Python networkx package | random-cubic_80.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_092.png?raw=true)
## [exact_093.gr](exact/exact_093.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
80 | 159 | 0.9 | planted | PACE 2020 | planted_rand_tree_80+80e.gr | 6 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_093.png?raw=true)
## [exact_094.gr](exact/exact_094.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
81 | 810 | 4.6 | named | https://github.com/freetdi/named-graphs | BrouwerHaemersGraph.gr | 57 | 

**Description:** Brouwer-Haemers graph, see https://en.wikipedia.org/wiki/Brouwer%E2%80%93Haemers_graph

![](vis/exact_094.png?raw=true)
## [exact_095.gr](exact/exact_095.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
82 | 100 | 0.6 | computer science | https://github.com/freetdi/CFGs | fuzix_devtty_keydecode.gr | 8 | 

**Description:** A control-flow graph of a C function

![](vis/exact_095.png?raw=true)
## [exact_096.gr](exact/exact_096.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
84 | 339 | 1.9 | biology | http://www.genome.jp/kegg/ | ncc00230.kgml.gr-1.gr | 17 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "ncc00230 Purine metabolism - Notothenia coriiceps (black rockcod)".

![](vis/exact_096.png?raw=true)
## [exact_097.gr](exact/exact_097.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
86 | 123 | 0.7 | social | http://networkrepository.com/ | ca-sandi_auths.gr | 7 | 

**Description:** A collaboration network

![](vis/exact_097.png?raw=true)
## [exact_098.gr](exact/exact_098.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
87 | 406 | 2.4 | social | Stanford Graph Base | david.gr | 18 | 

**Description:** David Coperfield graph from  Stanford Graph Base

![](vis/exact_098.png?raw=true)
## [exact_099.gr](exact/exact_099.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
88 | 133 | 1.0 | biology | http://networkrepository.com/ | ENZYMES_g8.gr | 13 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2020

![](vis/exact_099.png?raw=true)
## [exact_100.gr](exact/exact_100.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
90 | 135 | 0.8 | generated | Python networkx package | random-cubic_90.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_100.png?raw=true)
## [exact_101.gr](exact/exact_101.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
90 | 135 | 0.8 | named | https://github.com/freetdi/named-graphs | FosterGraph.gr | unknown | 

**Description:** Foster graph, see https://en.wikipedia.org/wiki/Foster_graph

![](vis/exact_101.png?raw=true)
## [exact_102.gr](exact/exact_102.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
92 | 104 | 0.6 | infrastructure | Open street maps | Szabda_Poland.gr | 9 | 

**Description:** Szabda (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_102.png?raw=true)
## [exact_103.gr](exact/exact_103.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
92 | 131 | 0.8 | infrastructure | Open street maps | Sypniewo_Poland.gr | 13 | 

**Description:** Sypniewo (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_103.png?raw=true)
## [exact_104.gr](exact/exact_104.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
92 | 285 | 1.6 | planted | PACE 2020 | planted_weighted_cycle_19_4+38v+38e.gr | 14 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_104.png?raw=true)
## [exact_105.gr](exact/exact_105.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
93 | 111 | 0.7 | infrastructure | Open street maps | Wysoka_Poland.gr | 9 | 

**Description:** Wysoka (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_105.png?raw=true)
## [exact_106.gr](exact/exact_106.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
94 | 118 | 0.7 | infrastructure | Open street maps | Płazowo_Poland.gr | 11 | 

**Description:** Płazowo (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_106.png?raw=true)
## [exact_107.gr](exact/exact_107.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
95 | 121 | 0.9 | biology | http://networkrepository.com/ | ENZYMES_g118.gr | 11 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2005

![](vis/exact_107.png?raw=true)
## [exact_108.gr](exact/exact_108.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
96 | 153 | 0.9 | generated | Python networkx package | waxman_100.gr | 16 | 

**Description:** A Waxman graph (a random geometric graph) generated using waxman_graph function from Python’s networkx package

![](vis/exact_108.png?raw=true)
## [exact_109.gr](exact/exact_109.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
98 | 109 | 0.7 | infrastructure | Open street maps | Miesiączkowo_Poland.gr | 9 | 

**Description:** Miesiączkowo (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_109.png?raw=true)
## [exact_110.gr](exact/exact_110.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
99 | 119 | 0.7 | infrastructure | Open street maps | Hornówek_Poland.gr | 10 | 

**Description:** Hornówek (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_110.png?raw=true)
## [exact_111.gr](exact/exact_111.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
100 | 384 | 2.3 | biology | http://www.genome.jp/kegg/ | onl00230.kgml.gr-0.gr | 18 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "onl00230 Purine metabolism - Oreochromis niloticus (Nile tilapia)".

![](vis/exact_111.png?raw=true)
## [exact_112.gr](exact/exact_112.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
100 | 1800 | 10.3 | named | https://github.com/freetdi/named-graphs | HallJankoGraph.gr | 83 | 

**Description:** Hall-Janko graph, see https://en.wikipedia.org/wiki/Hall%E2%80%93Janko_graph

![](vis/exact_112.png?raw=true)
## [exact_113.gr](exact/exact_113.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
102 | 113 | 0.6 | biology | http://www.genome.jp/kegg/ | rbb04024.kgml.gr-0.gr | 6 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "rbb04024 cAMP signaling pathway - Rhinopithecus bieti (black snub-nosed monkey)".

![](vis/exact_113.png?raw=true)
## [exact_114.gr](exact/exact_114.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
102 | 142 | 0.8 | biology | http://www.genome.jp/kegg/ | nni04010.kgml.gr-0.gr | 9 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "nni04010 MAPK signaling pathway - Nipponia nippon (crested ibis)".

![](vis/exact_114.png?raw=true)
## [exact_115.gr](exact/exact_115.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
102 | 144 | 0.9 | infrastructure | Open street maps | Wietrzychowice_Poland.gr | 12 | 

**Description:** Wietrzychowice (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_115.png?raw=true)
## [exact_116.gr](exact/exact_116.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
105 | 312 | 1.8 | planted | PACE 2020 | planted_weighted_cycle_23_4+46v+46e.gr | 14 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_116.png?raw=true)
## [exact_117.gr](exact/exact_117.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
105 | 441 | 3.2 | social | http://www-personal.umich.edu/~mejn/netdata/ | polbooks_105.gr | 24 | 

**Description:** Books about US politics: A network of books about US politics published around the time of the 2004 presidential election and sold by the online bookseller Amazon.com. Edges between books represent frequent copurchasing of books by the same buyers. The network was compiled by V. Krebs and is unpublished, but can found on Krebs' web site. Thanks to Valdis Krebs for permission to post these data on this web site.

![](vis/exact_117.png?raw=true)
## [exact_118.gr](exact/exact_118.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
110 | 219 | 1.3 | planted | PACE 2020 | planted_rand_tree_110+110e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_118.png?raw=true)
## [exact_119.gr](exact/exact_119.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
110 | 223 | 1.3 | biology | http://ginsim.org/models_repository | RodriguezJorge_TCR_Signalling_BooleanModel_17Jul2018.gr | 19 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the T cell receptor (TCR) signalling pathway and the Toll-like receptor (TLR5) signalling pathway. Reference: Rodr\'iguez-Jorge, Otoniel, Kempis-Calanis, Linda A., Abou-Jaoud\'e, Wassim, Guti\'errez-Reyna, Darely Y., Hernandez, C\'eline, Ramirez-Pliego, Oscar, Thomas-Chollier, Morgane, … (2019). Cooperation between t cell receptor and toll-like receptor 5 signaling for cd4+ t cell activation. Science Signaling, 12(577). http://dx.doi.org/10.1126/scisignal.aar3641

![](vis/exact_119.png?raw=true)
## [exact_120.gr](exact/exact_120.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
111 | 1029 | 6.0 | social | http://networkrepository.com/ | aves-songbird-social.gr | 38 | 

**Description:** an animal social network. A machine learning algorithm was applied to identify clusters of detections on feeders. Next, the network was generated based on patterns of co-occurrence by individuals in the same feeding events. Asociations between birds were defined using the simple ratio index. From: Adelman, James S., et al. "Feeder use predicts both acquisition and transmission of a contagious pathogen in a North American songbird." Proc. R. Soc. B. Vol. 282. No. 1815. The Royal Society, 2015.

![](vis/exact_120.png?raw=true)
## [exact_121.gr](exact/exact_121.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
112 | 168 | 1.0 | named | https://github.com/freetdi/named-graphs | Balaban11Cage.gr | unknown | 

**Description:** Balaban 11-cage graph, see https://en.wikipedia.org/wiki/Balaban_11-cage

![](vis/exact_121.png?raw=true)
## [exact_122.gr](exact/exact_122.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
112 | 425 | 3.0 | social | http://www-personal.umich.edu/~mejn/netdata/ | adjnoun_112.gr | unknown | 

**Description:** Adjacency network of common adjectives and nouns in the novel David Copperfield by Charles Dickens.

![](vis/exact_122.png?raw=true)
## [exact_123.gr](exact/exact_123.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
113 | 164 | 1.0 | infrastructure | Open street maps | Podzamcze_Poland.gr | 12 | 

**Description:** Podzamcze (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_123.png?raw=true)
## [exact_124.gr](exact/exact_124.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
113 | 168 | 1.1 | generated | Python networkx package | expander-chordal-cycle_113.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_124.png?raw=true)
## [exact_125.gr](exact/exact_125.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
115 | 161 | 1.0 | biology | http://www.genome.jp/kegg/ | amj04010.kgml.gr-0.gr | 10 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "amj04010 MAPK signaling pathway - Alligator mississippiensis (American alligator)".

![](vis/exact_125.png?raw=true)
## [exact_126.gr](exact/exact_126.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
117 | 146 | 0.9 | biology | http://www.genome.jp/kegg/ | mrt01130.kgml.gr-0.gr | 9 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of a map of pathways of Malassezia restricta.

![](vis/exact_126.png?raw=true)
## [exact_127.gr](exact/exact_127.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
119 | 147 | 0.9 | biology | http://www.genome.jp/kegg/ | mmu05200.kgml.gr-2.gr | 8 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "mmu05200 Pathways in cancer - Mus musculus (mouse)".

![](vis/exact_127.png?raw=true)
## [exact_128.gr](exact/exact_128.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
120 | 157 | 1.0 | infrastructure | Open street maps | Czerniewice_Poland.gr | 12 | 

**Description:** Czerniewice (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_128.png?raw=true)
## [exact_129.gr](exact/exact_129.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
120 | 240 | 1.5 | named | https://github.com/freetdi/named-graphs | BubbleSortGraph_5.gr | unknown | 

**Description:** Bubble sort graph for 5-permutations

![](vis/exact_129.png?raw=true)
## [exact_130.gr](exact/exact_130.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
120 | 479 | 2.9 | planted | PACE 2020 | planted_rand_tree_120+360e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_130.png?raw=true)
## [exact_131.gr](exact/exact_131.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
121 | 220 | 1.4 | generated | Python networkx package | grid_121.gr | unknown | 

**Description:** A grid graph generated using grid_graph function from Python’s networkx package

![](vis/exact_131.png?raw=true)
## [exact_132.gr](exact/exact_132.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
123 | 243 | 1.5 | named | https://github.com/freetdi/named-graphs | SierpinskiGasketGraph_5.gr | 12 | 

**Description:** Sierpinski Gasket graph of generation 5

![](vis/exact_132.png?raw=true)
## [exact_133.gr](exact/exact_133.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
123 | 409 | 2.5 | planted | PACE 2020 | planted_weighted_cycle_25_5+50v+50e.gr | 15 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_133.png?raw=true)
## [exact_134.gr](exact/exact_134.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
125 | 141 | 1.1 | biology | http://networkrepository.com/ | ENZYMES_g296.gr | 11 | 

**Description:** ENZYMES is a dataset of protein tertiary structures obtained from (Borgwardt et al., 2005)  consisting of 600 enzymes from the BRENDA enzyme database (Schomburg et al., 2004). Refernces: K. M. Borgwardt, C. S. Ong, S. Schoenauer, S. V. N. Vishwanathan, A. J. Smola, and H. P. Kriegel. Protein function prediction via graph kernels. Bioinformatics, 21(Suppl 1):i47–i56, Jun 2005. I. Schomburg, A. Chang, C. Ebeling, M. Gremse, C. Heldt, G. Huhn, and D. Schomburg. Brenda, the enzyme database: updates and major new developments. Nucleic Acids Research, 32D:431–433, 2009

![](vis/exact_134.png?raw=true)
## [exact_135.gr](exact/exact_135.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
126 | 189 | 1.2 | named | https://github.com/freetdi/named-graphs | Tutte12Cage.gr | unknown | 

**Description:** Tutte 12 cage graph, see https://en.wikipedia.org/wiki/Tutte_12-cage

![](vis/exact_135.png?raw=true)
## [exact_136.gr](exact/exact_136.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
126 | 315 | 1.9 | named | https://github.com/freetdi/named-graphs | OddGraph_5.gr | unknown | 

**Description:** Odd graph O_5, which is the Kneser graph KG(9,4)

![](vis/exact_136.png?raw=true)
## [exact_137.gr](exact/exact_137.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
126 | 4100 | 24.7 | computer science | http://archive.dimacs.rutgers.edu/pub/challenge/graph/benchmarks/color/ | zeroin.i.1.gr | 58 | 

**Description:** Problem based on register allocation for variables in real codes. Used in DIMACS Coloring Challenge 1992-1993. Contributor: Gary Lewandowski

![](vis/exact_137.png?raw=true)
## [exact_138.gr](exact/exact_138.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
127 | 189 | 1.2 | generated | Python networkx package | expander-chordal-cycle_127.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_138.png?raw=true)
## [exact_139.gr](exact/exact_139.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
128 | 266 | 1.7 | biology | http://ginsim.org/models_repository | RodriguezJorge_Merged_TCR_TLR5_Signalling_BooleanModel_15Jul2018.gr | 20 | 

**Description:** This is a gene regulatory network where the nodes are genes or cell states and edges correspond to regulatory interactions between pairs of genes or states, for example, if one gene is upregulated and reaches a certain level of activation, then another gene is upregulated as well. This network contains genes involved in the T cell receptor (TCR) signalling pathway and the Toll-like receptor (TLR5) signalling pathway. Reference: Rodr\'iguez-Jorge, Otoniel, Kempis-Calanis, Linda A., Abou-Jaoud\'e, Wassim, Guti\'errez-Reyna, Darely Y., Hernandez, C\'eline, Ramirez-Pliego, Oscar, Thomas-Chollier, Morgane, … (2019). Cooperation between t cell receptor and toll-like receptor 5 signaling for cd4+ t cell activation. Science Signaling, 12(577). http://dx.doi.org/10.1126/scisignal.aar3641

![](vis/exact_139.png?raw=true)
## [exact_140.gr](exact/exact_140.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
131 | 193 | 1.3 | generated | Python networkx package | expander-chordal-cycle_131.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_140.png?raw=true)
## [exact_141.gr](exact/exact_141.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
132 | 191 | 1.3 | infrastructure | Open street maps | Fabianki_Poland.gr | 13 | 

**Description:** Fabianki (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_141.png?raw=true)
## [exact_142.gr](exact/exact_142.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
133 | 167 | 1.0 | biology | http://www.genome.jp/kegg/ | cne01130.kgml.gr-0.gr | 9 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of a map of pathways of Cryptococcus neoformans var. neoformans JEC21.

![](vis/exact_142.png?raw=true)
## [exact_143.gr](exact/exact_143.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
138 | 447 | 2.8 | planted | PACE 2020 | planted_weighted_cycle_29_5+58v+58e.gr | 16 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_143.png?raw=true)
## [exact_144.gr](exact/exact_144.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
138 | 493 | 3.2 | social | Stanford Graph Base | anna.gr | 19 | 

**Description:** Anna Karenina graph from  Stanford Graph Base

![](vis/exact_144.png?raw=true)
## [exact_145.gr](exact/exact_145.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
140 | 279 | 1.8 | planted | PACE 2020 | planted_rand_tree_140+140e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_145.png?raw=true)
## [exact_146.gr](exact/exact_146.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
141 | 214 | 1.4 | generated | Python networkx package | waxman_150.gr | unknown | 

**Description:** A Waxman graph (a random geometric graph) generated using waxman_graph function from Python’s networkx package

![](vis/exact_146.png?raw=true)
## [exact_147.gr](exact/exact_147.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
142 | 196 | 1.3 | infrastructure | Open street maps | Łabiszyn-_Poland.gr | 13 | 

**Description:** Łabiszyn- (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_147.png?raw=true)
## [exact_148.gr](exact/exact_148.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
145 | 2512 | 15.0 | social | http://networkrepository.com/ | aves-wildbird-network-5.gr | 60 | 

**Description:** an animal social network. Groups were defined as individuals detected on the same nest-box during the same day, and co-memberships represented individuals that overlapped in nest-box exploration patterns during the same day. Networks were calculated from these group-by-individual matrices using the halfweight index. Each network represents social data collected for consecutive 6-day time window. From: Firth, Josh A., and Ben C. Sheldon. "Experimental manipulation of avian social structure reveals segregation is carried over across contexts." Proceedings of the Royal Society of London B: Biological Sciences 282.1802 (2015): 20142350.

![](vis/exact_148.png?raw=true)
## [exact_149.gr](exact/exact_149.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
146 | 250 | 1.6 | infrastructure | https://gitlab.com/rvb/mwcarp-ob | ob-Lpr-a-03.txt.gr | 17 | 

**Description:** a road network from the Ob generator by René van Bevern, Christian Komusiewicz, and Manuel Sorge (cities that are divided into several components by a river)

![](vis/exact_149.png?raw=true)
## [exact_150.gr](exact/exact_150.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
148 | 198 | 1.3 | infrastructure | Open street maps | Lubraniec_Poland.gr | 14 | 

**Description:** Lubraniec (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_150.png?raw=true)
## [exact_151.gr](exact/exact_151.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
150 | 599 | 3.8 | planted | PACE 2020 | planted_rand_tree_150+450e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_151.png?raw=true)
## [exact_152.gr](exact/exact_152.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
151 | 223 | 1.5 | generated | Python networkx package | expander-chordal-cycle_151.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_152.png?raw=true)
## [exact_153.gr](exact/exact_153.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
152 | 476 | 3.1 | planted | PACE 2020 | planted_weighted_cycle_31_5+62v+62e.gr | 14 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_153.png?raw=true)
## [exact_154.gr](exact/exact_154.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
157 | 3541 | 21.5 | computer science | http://archive.dimacs.rutgers.edu/pub/challenge/graph/benchmarks/color/ | zeroin.i.2.gr | 41 | 

**Description:** Problem based on register allocation for variables in real codes. Used in DIMACS Coloring Challenge 1992-1993. Contributor: Gary Lewandowski

![](vis/exact_154.png?raw=true)
## [exact_155.gr](exact/exact_155.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
160 | 240 | 1.6 | generated | Python networkx package | random-cubic_160.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_155.png?raw=true)
## [exact_156.gr](exact/exact_156.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
160 | 319 | 2.1 | planted | PACE 2020 | planted_rand_tree_160+160e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_156.png?raw=true)
## [exact_157.gr](exact/exact_157.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
163 | 195 | 1.3 | infrastructure | Open street maps | Czernikowo_Poland.gr | 11 | 

**Description:** Czernikowo (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_157.png?raw=true)
## [exact_158.gr](exact/exact_158.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
163 | 278 | 1.8 | infrastructure | https://gitlab.com/rvb/mwcarp-ob | ob-Lpr-b-03.txt.gr | unknown | 

**Description:** a road network from the Ob generator by René van Bevern, Christian Komusiewicz, and Manuel Sorge (cities that are divided into several components by a river)

![](vis/exact_158.png?raw=true)
## [exact_159.gr](exact/exact_159.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
165 | 522 | 3.4 | planted | PACE 2020 | planted_weighted_cycle_35_5+70v+70e.gr | 15 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_159.png?raw=true)
## [exact_160.gr](exact/exact_160.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
167 | 213 | 1.4 | biology | http://www.genome.jp/kegg/ | mrt01110.kgml.gr-0.gr | 12 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "mrt01110 Biosynthesis of secondary metabolites - Malassezia restricta".

![](vis/exact_160.png?raw=true)
## [exact_161.gr](exact/exact_161.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
167 | 328 | 2.2 | named | https://github.com/freetdi/named-graphs | WorldMap.gr | 13 | 

**Description:** Graph of all the countries, in which two countries are adjacent in the graph if they have a common boundary. From sage’s WorldMath() function.

![](vis/exact_161.png?raw=true)
## [exact_162.gr](exact/exact_162.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
170 | 218 | 1.5 | infrastructure | Open street maps | Osiek_Poland.gr | unknown | 

**Description:** Osiek (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_162.png?raw=true)
## [exact_163.gr](exact/exact_163.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
170 | 255 | 1.7 | generated | Python networkx package | random-cubic_170.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_163.png?raw=true)
## [exact_164.gr](exact/exact_164.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
170 | 679 | 4.3 | planted | PACE 2020 | planted_rand_tree_170+510e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_164.png?raw=true)
## [exact_165.gr](exact/exact_165.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
176 | 186 | 46.9 | infrastructure | https://github.com/daajoe/transit_graphs | GTFS_VBB_EndeApr_Dez2016.zip_metro.gr | 9 | 

**Description:** Transit graphs contributed by Johannes Fichte for PACE 2018

![](vis/exact_165.png?raw=true)
## [exact_166.gr](exact/exact_166.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
176 | 3973 | 24.8 | computer science | http://archive.dimacs.rutgers.edu/pub/challenge/graph/benchmarks/color/ | mulsol.i.5.gr | 45 | 

**Description:** Problem based on register allocation for variables in real codes. Used in DIMACS Coloring Challenge 1992-1993. Contributor: Gary Lewandowski

![](vis/exact_166.png?raw=true)
## [exact_167.gr](exact/exact_167.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
180 | 270 | 1.8 | generated | Python networkx package | random-cubic_180.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_167.png?raw=true)
## [exact_168.gr](exact/exact_168.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
180 | 359 | 2.5 | planted | PACE 2020 | planted_rand_tree_180+180e.gr | 7 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_168.png?raw=true)
## [exact_169.gr](exact/exact_169.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
181 | 253 | 1.8 | infrastructure | Open street maps | Nowogród_Poland.gr | unknown | 

**Description:** Nowogród (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_169.png?raw=true)
## [exact_170.gr](exact/exact_170.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
184 | 268 | 1.9 | infrastructure | Open street maps | Byczyna_Poland.gr | unknown | 

**Description:** Byczyna (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_170.png?raw=true)
## [exact_171.gr](exact/exact_171.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
191 | 283 | 2.0 | generated | Python networkx package | expander-chordal-cycle_191.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_171.png?raw=true)
## [exact_172.gr](exact/exact_172.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
195 | 342 | 2.3 | infrastructure | https://gitlab.com/rvb/mwcarp-ob | ob-Lpr-a-04.txt.gr | unknown | 

**Description:** a road network from the Ob generator by René van Bevern, Christian Komusiewicz, and Manuel Sorge (cities that are divided into several components by a river)

![](vis/exact_172.png?raw=true)
## [exact_173.gr](exact/exact_173.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
198 | 692 | 4.7 | planted | PACE 2020 | planted_weighted_cycle_39_5+78v+78e.gr | 16 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_173.png?raw=true)
## [exact_174.gr](exact/exact_174.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
199 | 265 | 1.9 | infrastructure | Open street maps | Barcin-_Poland.gr | 14 | 

**Description:** Barcin- (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_174.png?raw=true)
## [exact_175.gr](exact/exact_175.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
200 | 300 | 2.1 | generated | Python networkx package | random-cubic_200.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_175.png?raw=true)
## [exact_176.gr](exact/exact_176.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
200 | 799 | 5.4 | planted | PACE 2020 | planted_rand_tree_200+600e.gr | 8 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_176.png?raw=true)
## [exact_177.gr](exact/exact_177.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
204 | 248 | 1.6 | biology | http://www.genome.jp/kegg/ | olu01110.kgml.gr-0.gr | 9 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "olu01110 Biosynthesis of secondary metabolites - Ostreococcus lucimarinus".

![](vis/exact_177.png?raw=true)
## [exact_178.gr](exact/exact_178.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
210 | 2520 | 17.2 | named | https://github.com/freetdi/named-graphs | JohnsonGraph_10_4.gr | unknown | 

**Description:** Johnson J(10,4) graph, see https://en.wikipedia.org/wiki/Johnson_graph

![](vis/exact_178.png?raw=true)
## [exact_179.gr](exact/exact_179.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
212 | 257 | 1.8 | infrastructure | Open street maps | Mielnik_Poland.gr | unknown | 

**Description:** Mielnik (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_179.png?raw=true)
## [exact_180.gr](exact/exact_180.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
214 | 785 | 5.4 | planted | PACE 2020 | planted_weighted_cycle_41_5+82v+82e.gr | 17 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_180.png?raw=true)
## [exact_181.gr](exact/exact_181.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
223 | 265 | 1.8 | biology | http://www.genome.jp/kegg/ | bpg01110.kgml.gr-0.gr | 10 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "bpg01110 Biosynthesis of secondary metabolites - Bathycoccus prasinos".

![](vis/exact_181.png?raw=true)
## [exact_182.gr](exact/exact_182.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
225 | 771 | 5.3 | planted | PACE 2020 | planted_weighted_cycle_45_5+90v+90e.gr | 17 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_182.png?raw=true)
## [exact_183.gr](exact/exact_183.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
231 | 3465 | 23.9 | named | https://github.com/freetdi/named-graphs | CameronGraph.gr | unknown | 

**Description:** Cameron graph, see  see https://www.win.tue.nl/~aeb/graphs/Cameron.html

![](vis/exact_183.png?raw=true)
## [exact_184.gr](exact/exact_184.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
244 | 290 | 2.0 | biology | http://www.genome.jp/kegg/ | mpp01110.kgml.gr-0.gr | 10 | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "mpp01110 Biosynthesis of secondary metabolites - Micromonas pusilla".

![](vis/exact_184.png?raw=true)
## [exact_185.gr](exact/exact_185.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
276 | 1187 | 8.4 | planted | PACE 2020 | planted_weighted_cycle_50_6+100v+150e.gr | 19 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_185.png?raw=true)
## [exact_186.gr](exact/exact_186.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
280 | 360 | 2.6 | infrastructure | Open street maps | Wolica_Poland.gr | 14 | 

**Description:** Wolica (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_186.png?raw=true)
## [exact_187.gr](exact/exact_187.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
300 | 450 | 3.2 | generated | Python networkx package | random-cubic_300.gr | unknown | 

**Description:** A random cubic graph generated using random_regular_graph function from Python’s networkx package

![](vis/exact_187.png?raw=true)
## [exact_188.gr](exact/exact_188.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
300 | 508 | 3.3 | biology | https://thebiogrid.org/ | S1_Table.gr | 13 | 

**Description:** This is a protein-interaction network containing interactions determined using tandem-affinity purification. Data gathered by Cloutier, P. et al., R2tp/prefoldin-like component ruvbl1/ruvbl2 directly interacts with znhit2 to regulate assembly of u5 small nuclear ribonucleoprotein. Nature Communications, 8(1), 15615, 2007. http://dx.doi.org/10.1038/ncomms15615

![](vis/exact_188.png?raw=true)
## [exact_189.gr](exact/exact_189.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
300 | 1199 | 8.5 | planted | PACE 2020 | planted_rand_tree_300+900e.gr | 8 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_189.png?raw=true)
## [exact_190.gr](exact/exact_190.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
352 | 440 | 3.2 | infrastructure | Open street maps | Kcynia_Poland.gr | unknown | 

**Description:** Kcynia (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_190.png?raw=true)
## [exact_191.gr](exact/exact_191.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
439 | 873 | 6.4 | generated | Python networkx package | waxman_450.gr | unknown | 

**Description:** A Waxman graph (a random geometric graph) generated using waxman_graph function from Python’s networkx package

![](vis/exact_191.png?raw=true)
## [exact_192.gr](exact/exact_192.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
441 | 1638 | 12.1 | generated | Python networkx package | expander-margulis_441.gr | unknown | 

**Description:** A  Margulis-Gabber-Galil undirected graph (8-regular expander with multiple edges removed) on n^2 nodes. Generated using margulis_gabber_galil_graph from Python’s networkx package

![](vis/exact_192.png?raw=true)
## [exact_193.gr](exact/exact_193.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
449 | 2213 | 16.3 | planted | PACE 2020 | planted_weighted_cycle_75_7+150v+262e.gr | 24 | 

**Description:** A cycle of cliques of random cycles with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_193.png?raw=true)
## [exact_194.gr](exact/exact_194.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
450 | 1799 | 12.9 | planted | PACE 2020 | planted_rand_tree_450+1350e.gr | 8 | 

**Description:** A random tree with some added random edges that go bottom-up the optimal treedepth decomposition

![](vis/exact_194.png?raw=true)
## [exact_195.gr](exact/exact_195.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
451 | 587 | 4.3 | infrastructure | Open street maps | SanMarino.gr | unknown | 

**Description:** San Marino: a road network downloaded from open street maps using osmnx Python package

![](vis/exact_195.png?raw=true)
## [exact_196.gr](exact/exact_196.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
457 | 684 | 5.1 | generated | Python networkx package | expander-chordal-cycle_457.gr | unknown | 

**Description:** A cycle graph on p nodes with chords joining each vertex x to its inverse modulo p. This graph is a (mildly explicit) 3-regular expander. Generated using chordal_cycle_graph function from Python’s networkx package

![](vis/exact_196.png?raw=true)
## [exact_197.gr](exact/exact_197.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
469 | 614 | 4.4 | biology | http://www.genome.jp/kegg/ | uma01100.kgml.gr-0.gr | unknown | 

**Description:** This is a graph derived from a biological pathway map. The nodes are genes (also representing their associated proteins), enzymes, or chemical compounds, and there is an edge between two nodes if they influence each other (such as a gene modifying the expression of another gene) or take part in a common chemical reaction. This graph is a connected part of the map "uma01100 Metabolic pathways - Ustilago maydis".

![](vis/exact_197.png?raw=true)
## [exact_198.gr](exact/exact_198.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
479 | 638 | 4.8 | infrastructure | Open street maps | Czersk_Poland.gr | unknown | 

**Description:** Czersk (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_198.png?raw=true)
## [exact_199.gr](exact/exact_199.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
491 | 645 | 4.7 | biology | https://thebiogrid.org/ | Bandyopadhyay2010.gr | 17 | 

**Description:** This is a protein-interaction network between human mitogen-activated protein kinase (MAPK)-related proteins and other cellular machinery. The nodes are the proteins and edges represent that the proteins bind to each other as detected by using "yeast two-hybrid screening". Data gathered by Bandyopadhyay, S. et al., A human map kinase interactome. Nature Methods, 7(10), 801–805, 2010. http://dx.doi.org/10.1038/nmeth.1506

![](vis/exact_199.png?raw=true)
## [exact_200.gr](exact/exact_200.gr)
vertices   | edges   | size (kB) | type | origin | source filename | treedepth (if known) | 
 --- | --- | --- | --- | --- | --- | --- |
498 | 702 | 5.3 | infrastructure | Open street maps | Terespol_Poland.gr | unknown | 

**Description:** Terespol (Poland): a road network downloaded from open street maps using osmnx Python package

![](vis/exact_200.png?raw=true)
